string file_name
---
bool result
